cupyx/scipy/special/digamma.py

```diff
@@ -0,0 +1,199 @@
+import cupy
+from cupy import core
+
+
+_digamma_kernel = None
+
+polevl_definition = '''
+static __device__ double polevl(double x, double coef[], int N)
+{
+    double ans;
+    int i;
+    double *p;
+
+    p = coef;
+    ans = *p++;
+    i = N;
+
+    do
+    ans = ans * x + *p++;
+    while (--i);
+
+    return (ans);
+}
+'''
+
+
+psi_definition = '''
+static __device__ double A[] = {
```

Comment:
Make it const. Ditto for other static variables. Also it may be worth trying `__const__` instead of `__device__ const` (I have not used it, so it's better to measure the performance).